packages/backend/src/models/UserModel.test.ts

```diff
@@ -0,0 +1,355 @@
+import { subject } from '@casl/ability';
+import { UserModel } from './UserModel';
+import {
+    adminOrgProfile,
+    adminProjectProfile,
+    conditions as defaultConditions,
+    orgProfile,
+    projectProfile,
+} from './UserModel.mock';
+
+describe('Project member permissions', () => {
+    let ability = UserModel.mergeUserAbilities(orgProfile, [projectProfile]);
+    let conditions = defaultConditions;
+    describe('when user is an org viewer and project viewer', () => {
+        beforeEach(() => {
+            ability = UserModel.mergeUserAbilities(orgProfile, [
+                projectProfile,
+            ]);
+        });
+
+        it('can view org and project', async () => {
+            expect(
+                ability.can('view', subject('SavedChart', { ...conditions })),
+            ).toEqual(true);
+            expect(
+                ability.can('view', subject('Dashboard', { ...conditions })),
+            ).toEqual(true);
+            expect(
+                ability.can('view', subject('Project', { ...conditions })),
+            ).toEqual(true);
+        });
+
+        it('cannot view another org or another project', async () => {
+            conditions = {
+                organizationUuid: 'another-org',
+                projectUuid: 'another-project',
+            };
+            expect(
+                ability.can('view', subject('SavedChart', { ...conditions })),
+            ).toEqual(false);
+            expect(
+                ability.can('view', subject('Dashboard', { ...conditions })),
+            ).toEqual(false);
+            expect(
+                ability.can('view', subject('Project', { ...conditions })),
+            ).toEqual(false);
+        });
+
+        it('cannot view another org or another project', async () => {
```

Comment:
Duplicated `cannot view another org or another project`. Can we add a test with correct org but incorrect project ?